opengrok-indexer/src/main/java/org/opengrok/indexer/authorization/AuthControlFlag.java

```diff
@@ -48,13 +48,17 @@
      */
     REQUISITE("requisite"),
     /**
-     * If such a plugin succeeds and no prior required plugin has failed the
-     * authorization framework returns success to the application immediately
-     * without calling any further plugins in the stack. A failure of a
-     * sufficient plugin is ignored and processing of the plugin list continues
-     * unaffected.
+     * If such a plugin succeeds and no prior required plugin has failed,
+     * the authorization framework returns success immediately
+     * without calling any further plugins in the stack.
+     * A failure of a sufficient plugin is ignored and processing of the stack continues unaffected.
      */
-    SUFFICIENT("sufficient");
+    SUFFICIENT("sufficient"),
+    /**
+     * The failure is returned only if stack traversal finished and
+     * there was no prior result recorded.
```

Comment:
maybe worth noting that: A success of a sufficient plugin is ignored and processing of the stack continues unaffected.

Reply:
added